flume-ng-sources/flume-kafka-source/pom.xml

```diff
@@ -83,9 +83,8 @@
     </dependency>
     <dependency>
       <groupId>org.apache.kafka</groupId>
-      <artifactId>kafka_2.10</artifactId>
-      <classifier>test</classifier>
-      <scope>test</scope>
```

Comment:
test scope should remain for this - we only need full Kafka for the tests, right?